src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +202,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        interpretInputAndExecute();
+    }
+
+	private static void interpretInputAndExecute() {
+		while (true) {
```

Comment:
Reformat according to coding style after everytime you had made the code change. use eclipse shortcut.